core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala

```diff
@@ -102,7 +102,7 @@ private[ui] class ExecutorTable(stageId: Int, stageAttemptId: Int, parent: Stage
 
     listener.stageIdToData.get((stageId, stageAttemptId)) match {
       case Some(stageData: StageUIData) =>
-        stageData.executorSummary.toSeq.sortBy(_._1).map { case (k, v) =>
+        stageData.executorSummary.toSeq.sortBy(_._1.toLong).map { case (k, v) =>
```

Comment:
Hm, that's not a great argument to me, as it means "you must implement sorting logic twice, always". It's not possible to trigger the JS after the table loads? in any event, it may not be so hard to sort both places, sure. But it's a weird sort: by number for things that can be numbers but then alphabetically otherwise. That has to be implemented two places. It may prove to be too much hassle, as in fact, executors aren't really numbers but only usually appear to be